Detector/DetFCChhBaseline1/compact/FCChh_DectMaster.xml

```diff
@@ -31,7 +31,7 @@
   <include ref="./FCChh_Shielding.xml" />
 
 
-  <include ref="../../DetFCChhTrackerSimple/compact/Tracker.xml" />
+  <include ref="../../DetFCChhTrackerTkLayout/compact/Tracker.xml" />
```

Comment:
I don't think it is still needed, for most purposes `Detector/DetFCChhBaseline1/compact/estimates/FCChh_TrackerMaterial.xml` is probably more useful. I'm working on a simplified version of the TkLayoutTracker that can be used in the reco also. I'll remove it.